policy/testdata/policybundle-deps.mql.yaml

```diff
@@ -1,57 +1,48 @@
 owner_mrn: //captain.api.mondoo.app/spaces/adoring-moore-542492
 policies:
-- mrn: //assets.api.mondoo.app/spaces/adoring-moore-542492/assets/1dKBiOi5lkI2ov48plcowIy8WEl
-  version: 1.0.0
-  asset_filters:
-    platform.name == "debian":
-      query: platform.name == "debian"
-  specs:
-  - policies:
-      //captain.api.mondoo.app/spaces/adoring-moore-542492: null
-- mrn: //captain.api.mondoo.app/spaces/adoring-moore-542492
-  owner_mrn: //captain.api.mondoo.app/spaces/adoring-moore-542492
-  version: 1.0.0
-  asset_filters:
-    platform.name == "debian":
-      query: platform.name == "debian"
-  specs:
-  - policies:
-      //policy.api.mondoo.app/policies/debian-10-level-1-server: null
-    scoring_queries:
-      //policy.api.mondoo.app/queries/1.6.1-xd-nx-support-enabled:
-        action: 2
-        id: //policy.api.mondoo.app/queries/1.6.1-xd-nx-support-enabled
-- mrn: //policy.api.mondoo.app/policies/debian-10-level-1-server
-  owner_mrn: //policy.api.mondoo.app
-  name: Debian Linux 10 Benchmark Level 1 - Server Profile
-  version: 1.0.0
-  is_public: true
-  asset_filters:
-    platform.name == "debian":
-      query: platform.name == "debian"
-  specs:
-  - asset_filter:
-      query: platform.name == "debian"
-    scoring_queries:
-      //policy.api.mondoo.app/queries/1.1.1.1-mounting-freevxfs-filesystems-disabled:
-        id: //policy.api.mondoo.app/queries/1.1.1.1-mounting-freevxfs-filesystems-disabled
-      //policy.api.mondoo.app/queries/1.1.1.2-mounting-jffs2-filesystems-disabled:
-        id: //policy.api.mondoo.app/queries/1.1.1.2-mounting-jffs2-filesystems-disabled
-      //policy.api.mondoo.app/queries/1.1.1.3-mounting-hfs-filesystems-disabled:
-        id: //policy.api.mondoo.app/queries/1.1.1.3-mounting-hfs-filesystems-disabled
+  - computed_filters: platform.name == "debian"
```

Comment:
i feel like `computed_filters` was an internal thing and not to be set explicitly

Reply:
I have not changed those values, I just reformatted it. It is a great call to remove those though.